Games/types/Mafia/items/Flier.js

```diff
@@ -16,6 +17,10 @@ module.exports = class Flier extends Item {
                 states: ["Night"],
                 flags: ["group", "speech", "voting", "mustAct", "noVeg"],
                 inputType: "boolean",
+                passiveDead: true,
```

Comment:
should dead people be allowed to see what is spoken in the party?
both ways work for me.

with this implementation, we're treating party as the night version of village core